project/src/pages/my-list/my-list.tsx

```diff
@@ -1,14 +1,29 @@
 
 import { Film } from '../../types/film/film';
 import FilmsList from '../../components/films-list/films-list';
+import Footer from '../../components/footer/footer';
+import Header from '../../components/header/header';
 
-function MyList(films: Film[]): JSX.Element {
+type MyListProps = {
+  films: Film[]
+}
+
+function MyList({films}: MyListProps): JSX.Element {
   return (
-    <section className="catalog">
-      <h2 className="catalog__title visually-hidden">Catalog</h2>
+    <div className="user-page">
+      <Header
+        heading={'My list'}
```

Comment:
Если строка то можно просто так 
```suggestion
        heading="My list"
```